.github/workflows/ci_cd.yml

```diff
@@ -102,6 +96,26 @@ jobs:
           just precommit
           just lint
 
+  add-stack-label:
+    name: Add stack label
+    runs-on: ubuntu-latest
+    needs:
+      - get-changes
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Add backend stack label
+        uses: actions-ecosystem/action-add-labels@v1
+        if: ${{ needs.get-changes.outputs.api == 'true' || needs.get-changes.outputs.ingestion_server == 'true' }}
+        with:
+          labels: "🧱 stack: backend"
```

Comment:
The ingestion server and the API have different labels. "🧱 stack: api" and "🧱 stack: ingestion server". To the best of my knowledge, the "🧱 stack: backend" label was a placeholder when importing issues for both projects from the `openverse-api` repo.

Reply:
Yes, I added the "backend" stack label to make the issue import easier. It would be nice to review all the "backend" issues and apply the correct stack some time in the future.

Reply:
I see, I think when I made this the `backend` label was the only one available 🤔 Perhaps not though, I'll change this so it references the right one!